site-configs.types.ts

```diff
@@ -0,0 +1,11 @@
+import type { SiteConfig as SiteConfigLibrary, SiteConfigPrivate, SiteConfigPublic } from "@comet/cli";
```

Comment:
I'm not convinced about having wrappers in the application that have the same name as the library anymore. This repeatedly leads to importing the wrong types, functions etc. Could we use other names if we always want to provide types in the application?

For example, `SiteConfigPrivate` could be `ExtractPrivateSiteConfig`:

```ts
type PrivateSiteConfig = ExtractPrivateSiteConfig<SiteConfig>;
```

Reply:
good idea